utils/qovery.go

```diff
@@ -1402,6 +1424,13 @@ func IsTerminalState(state qovery.StateEnum) bool {
 		strings.HasSuffix(string(state), "ERROR")
 }
 
+func IsTerminalClusterState(state qovery.ClusterStateEnum) bool {
```

Comment:
Maybe having invalid credentials should be considered as a terminal state ?

Reply:
yes I didn't see the  strings.HasSuffix(string(state), "ERROR")